library/signed/src/Tests.qs

```diff
@@ -8,47 +8,24 @@ import Measurement.MeasureSignedInteger;
 /// This entrypoint runs tests for the signed integer library.
 operation Main() : Unit {
     UnsignedOpTests();
-    MeasureSignedIntTests();
+    Fact(Qtest.Operations.CheckAllTestCases(MeasureSignedIntTests()), "SignedInt tests failed");
     SignedOpTests();
 
 }
 
-operation MeasureSignedIntTests() : Unit {
-    use a = Qubit[4];
-
-    // 0b0001 == 1
-    X(a[0]);
-    let res = MeasureSignedInteger(a, 4);
-    Fact(res == 1, $"Expected 1, received {res}");
-
-    // 0b1111 == -1
-    X(a[0]);
-    X(a[1]);
-    X(a[2]);
-    X(a[3]);
-    let res = MeasureSignedInteger(a, 4);
-    Fact(res == -1, $"Expected -1, received {res}");
-
-    // 0b01000 == 8
-    use a = Qubit[5];
-    X(a[3]);
-    let res = MeasureSignedInteger(a, 5);
-    Fact(res == 8, $"Expected 8, received {res}");
-
-    // 0b11110 == -2
-    X(a[1]);
-    X(a[2]);
-    X(a[3]);
-    X(a[4]);
-    let res = MeasureSignedInteger(a, 5);
-    Fact(res == -2, $"Expected -2, received {res}");
-
-    // 0b11000 == -8
-    X(a[3]);
-    X(a[4]);
-    let res = MeasureSignedInteger(a, 5);
-    Fact(res == -8, $"Expected -8, received {res}");
-
+function MeasureSignedIntTests() : (String, Int, (Qubit[]) => (), (Qubit[]) => Int, Int)[] {
+    [
+        ("0b0001 == 1", 4, (qs) => X(qs[0]), (qs) => MeasureSignedInteger(qs, 4), 1),
```

Comment:
@sezna , it seems these tests should not pass since the signature of `QTest.Operations.TestCases` was changed. Do the current library tests get checked in CI or am I missing something here.

Reply:
Hey Manvi, I'm not seeing an issue here. The function call here [does indeed use the latest api](https://github.com/microsoft/qsharp/pull/2013/files/85beeeaba7734d37a48e2f750c0f5e0ea3c84d4c#diff-fa6f0fda29e55d52d8254ab813dcda39094b02928370e841ea741c526e3d5338R11).

Reply:
@sezna, I realized it now, that this still accepts a tuple containing 5 values. I think I got confused by 2 factors:
- **Incorrect [doc string](https://github.com/microsoft/qsharp/blob/main/library/qtest/src/Operations.qs#L14-L22)** : this seems to be copy paste of the functions namespace. Could you please fix it?
- **Unnecessary braces around (Qubit[]) in method signature**: Its written as `(Qubit[])=> ()` here and in `Operations.qs`. This can be written as `Qubit[] => ()`.